src/Prelude.hs

```diff
@@ -11,6 +11,7 @@ module Prelude
        , module Web
 
        , WithLog
+       , (\\)
```

Comment:
Let's not add this operator to `Prelude`. It's better to export it explicitly in the module... Usually we add things to `Prelude` which are:

1. Supposed to be used in at least half of the modules.
2. Have unique enough names to not conflict with anything else.